pipeline/transform.py

```diff
@@ -91,9 +89,9 @@ def get_earthquake_data(data: dict) -> dict:
     earthquake_id = get_earthquake_id(data)  # validate_earthquake_naming
     alert = get_earthquake_property(data, 'alert')  #validate_property
     status = get_earthquake_property(data, 'status')  # validate_property
-    network = get_earthquake_property(data, 'net')  # validate_property
-    magtype = get_earthquake_property(data, 'magType')  # validate_property
-    earthquake_type = get_earthquake_property(data, 'type')  # validate_property
+    network = get_earthquake_property(data, 'net')  # validate_network
+    magtype = get_earthquake_property(data, 'magType')  # validate_types
+    earthquake_type = get_earthquake_property(data, 'type')  # validate_types
```

Comment:
I notice you have specified in the comments the values being validated, but only for network, magtype and earthquake type. Is there a reason for this, or could you add this detail in the other comments you have made in lines 90-97?

Reply:
All the values are validated, I've added the comments on each value so its easier to find how its being validated